commands/faas.go

```diff
@@ -5,12 +5,15 @@ package commands
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/spf13/cobra"
 )
 
 const defaultGateway = "http://localhost:8080"
+
 const defaultNetwork = "func_functions"
+const defaultYaml = "stack.yml"
```

Comment:
Would you be open to having two const variables here:

`defaultYml = "stack.yml"`
`defaultYaml = "stack.yaml"`

Then have a `else if ` block for `defaultYml` on line 59

Reply:
Hmm.. I think we can assume `yml`only for a default vs `yaml`. Having had a quick google "YML" as an extension is the winner for usage.